patches/v2024.6.2d.patch

```diff
@@ -0,0 +1,1996 @@
++            />
+             <button type="button" bitIconButton bitSuffix bitPasswordInputToggle></button>
+             <bit-hint>{{ "sendPasswordDesc" | i18n }}</bit-hint>
+           </bit-form-field>
+diff --git a/apps/web/src/app/vault/individual-vault/vault-filter/components/organization-options.component.ts b/apps/web/src/app/vault/individual-vault/vault-filter/components/organization-options.component.ts
+index 8dd63e62dd..1a1e45cc35 100644
+--- a/apps/web/src/app/vault/individual-vault/vault-filter/components/organization-options.component.ts
++++ b/apps/web/src/app/vault/individual-vault/vault-filter/components/organization-options.component.ts
+@@ -10,6 +10,7 @@ import { PolicyService } from "@bitwarden/common/admin-console/abstractions/poli
+ import { PolicyType } from "@bitwarden/common/admin-console/enums";
+ import { Organization } from "@bitwarden/common/admin-console/models/domain/organization";
+ import { Policy } from "@bitwarden/common/admin-console/models/domain/policy";
++import { UserVerificationService } from "@bitwarden/common/auth/abstractions/user-verification/user-verification.service.abstraction";
+ import { I18nService } from "@bitwarden/common/platform/abstractions/i18n.service";
+ import { LogService } from "@bitwarden/common/platform/abstractions/log.service";
+ import { PlatformUtilsService } from "@bitwarden/common/platform/abstractions/platform-utils.service";
+@@ -48,6 +49,7 @@ export class OrganizationOptionsComponent implements OnInit, OnDestroy {
+     private userDecryptionOptionsService: UserDecryptionOptionsServiceAbstraction,
+     private dialogService: DialogService,
+     private resetPasswordService: OrganizationUserResetPasswordService,
++    private userVerificationService: UserVerificationService,
+   ) {}
+ 
+   async ngOnInit() {
+@@ -155,6 +157,7 @@ export class OrganizationOptionsComponent implements OnInit, OnDestroy {
+         this.i18nService,
+         this.syncService,
+         this.logService,
++        this.userVerificationService,
+       );
+     } else {
+       // Remove reset password
+diff --git a/apps/web/src/index.html b/apps/web/src/index.html
+index c3a2c03ed9..ad807d3dd2 100644
+--- a/apps/web/src/index.html
++++ b/apps/web/src/index.html
+@@ -5,19 +5,20 @@
+     <meta name="viewport" content="width=1010" />
+     <meta name="theme-color" content="#175DDC" />
+ 
+-    <title page-title>Bitwarden Web Vault</title>
++    <title page-title>Vaultwarden Web</title>
+ 
+     <link rel="apple-touch-icon" sizes="180x180" href="images/icons/apple-touch-icon.png" />
+     <link rel="icon" type="image/png" sizes="32x32" href="images/icons/favicon-32x32.png" />
+     <link rel="icon" type="image/png" sizes="16x16" href="images/icons/favicon-16x16.png" />
+     <link rel="mask-icon" href="images/icons/safari-pinned-tab.svg" color="#175DDC" />
+     <link rel="manifest" href="manifest.json" />
++    <link rel="stylesheet" href="<%= htmlWebpackPlugin.options.vw_css %>" />
```

Comment:
Instead of excluding index.html from being handled, you can just use [`<!-- webpackIgnore: true -->`](https://webpack.js.org/loaders/html-loader/#disable-url-resolving-using-the----webpackignore-true----comment) to disable the source handling for this file:
```suggestion
+    <!-- webpackIgnore: true -->
+    <link rel="stylesheet" href="css/vaultwarden.css" />
```

Reply:
Ah! If that then still won't remove it that would be a better option, thanks!

Reply:
[This is the generated index.html (saved as .txt so it can be uploaded)](https://github.com/user-attachments/files/17001078/index.txt) when I tested it, so the line is untouched.

Reply:
Cool